src/include/generation/generate_c.h

```diff
@@ -93,7 +93,6 @@ int generate_c_code(
 
 int generate_c_include_prelude(FILE* output_file) {
 	fputs("#include <assert.h>\n", output_file);
-	fputs("#include <math.h>\n", output_file);
```

Comment:
I do not want to include this by default, as I intend to write a math standard library for Daybreak very soon. If needed, this can now be imported with
```nim
importc "math.h"
```